docs/api/datasets.rst

```diff
@@ -93,6 +93,12 @@ iNaturalist
 
 .. autoclass:: INaturalist
 
+LandCover.ai
+^^^^^^^^^^^^
+
+.. autoclass:: LandCoverAI
```

Comment:
RtD tests are failing because this class is documented twice (Sphinx doesn't know which place to link to). Maybe only document it here and remove the reference below.